domainbed/algorithms.py

```diff
@@ -4,15 +4,19 @@
 import torch.nn as nn
 import torch.nn.functional as F
 import torch.autograd as autograd
+from torch.autograd import Variable
 
 import copy
 import numpy as np
+from collections import defaultdict
 
 from domainbed import networks
-from domainbed.lib.misc import random_pairs_of_minibatches
+from domainbed.lib.misc import random_pairs_of_minibatches, ParamDict
 
 ALGORITHMS = [
     'ERM',
+    'Fish',
+    'FishSmooth',
```

Comment:
Hi Yuge! Thank you for your PR. We are currently offering only one version per algorithm. Would you mind removing one fish? This is due to computational budget and to give everyone the same change. You can encapsulate the behaviour of both models using an extra hyper-parameter, if you like.